rocAL/rocAL_pybind/examples/rocAL_api_caffe2_reader.py

```diff
@@ -72,22 +72,22 @@ def main():
     with pipe:
         if _rocal_bbox:
             jpegs, labels, bboxes = fn.readers.caffe2(
-                path=image_path, bbox=_rocal_bbox, random_shuffle=True)
+                path=image_path, bbox=_rocal_bbox)
```

Comment:
Why is the variable rocal_bbox conatins an "_"
Make sure we follow the same pattern everywhere when we declare arguments. Remove the underscore from rocal_bbox & rocal_cpu
    image_path = args.image_dataset_path
    _rocal_cpu = False if args.rocal_gpu else True
    batch_size = args.batch_size
    _rocal_bbox = False if args.classification else True
    num_threads = args.num_threads
    local_rank =  args.local_rank
    random_seed = args.seed
    display = True if args.display else False
    device = "gpu" if args.rocal_gpu else "cpu"
    num_classes = len(next(os.walk(image_path))[1])

Reply:
The changes were only done in this example regarding the tensor pipeline conversion. Variable and function names are left as it is.